vortexor/Cargo.toml

```diff
@@ -19,6 +19,8 @@ crate-type = ["lib"]
 name = "solana_vortexor"
 
 [features]
+default = ["agave-unstable-api"]
```

Comment:
Was this intended?

Reply:
think i just missed it

Reply:
nope. either need this or to remove the deprecation warning from the lib. this is another multi-target abomination